ecs-init/volumes/ecs_volume_plugin.go

```diff
@@ -310,14 +313,17 @@ func (a *AmazonECSVolumePlugin) List() (*volume.ListResponse, error) {
 func (a *AmazonECSVolumePlugin) Get(r *volume.GetRequest) (*volume.GetResponse, error) {
 	a.lock.RLock()
 	defer a.lock.RUnlock()
-	_, ok := a.volumes[r.Name]
+	vol, ok := a.volumes[r.Name]
 	if !ok {
 		return nil, fmt.Errorf("volume %s not found", r.Name)
 	}
-	vol := &volume.Volume{
-		Name: r.Name,
+	resp := &volume.Volume{
+		Name:       r.Name,
+		Mountpoint: vol.Path,
+		CreatedAt:  vol.CreatedAt,
 	}
-	return &volume.GetResponse{Volume: vol}, nil
+	seelog.Infof("Returning volume information: %v", resp)
```

Comment:
how frequently are we expecting to call this function? Do we need to log the full response every time?

Reply:
on similar note, maybe consider extracting minimum necessary info from resp to log?

Reply:
just tested. this is printed twice for mounting volume to a container, docker stop container and docker volume inspect. This could become noisy if we are printing the full response. But this could still be valuable when debugging failures. I'll change it to it prints just the volume name.